llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9289,7 +9288,7 @@ void VPReductionRecipe::execute(VPTransformState &State) {
     else
       NextInChain = State.Builder.CreateBinOp(
           (Instruction::BinaryOps)RdxDesc.getOpcode(Kind), NewRed, PrevInChain);
-    State.set(this, NextInChain, Part);
+    State.set(this, NextInChain, VPIteration(Part, 0));
```

Comment:
```suggestion
    State.set(this, NextInChain, Part, true /* scalar */);
```
same API of `get` should apply to `set` as well.